lib/validators/specResolver.js

```diff
@@ -590,6 +601,23 @@ class SpecResolver {
     return Promise.resolve(self);
   }
 
+  /**
+   * Models a default response as a Cloud Error if none is specified in the api spec.
+   */
+  modelImplicitDefaultResponse() {
+    let self = this;
+    let spec = self.specInJson;
+    for (let pathObj of utils.getValues(spec.paths)) {
+      for (let operation of utils.getValues(pathObj)) {
+
+        // going through responses
+        if (operation.responses && !operation.responses.default) {
+          operation.responses.default = utils.CloudError;
```

Comment:
```javascript
if (!spec.definitions.CloudError) {
  spec.definitions.CloudErrorWrapper = utils.CloudErrorWrapper;
  spec.definitions.CloudError = utils.CloudError;
}
```

Reply:
@amarzavery so you prefer to define the cloud error in the definitions and then just make a ref to it ? that works I think